SCClassLibrary/Common/Control/Server.sc

```diff
@@ -376,45 +412,48 @@ Server {
 	}
 
 	newNodeAllocators {
-		nodeAllocator = NodeIDAllocator(clientID, options.initialNodeID)
+		nodeAllocator = nodeAllocClass.new(clientID,
+			options.initialNodeID, this.numUsers)
```

Comment:
My comment above means all those `this.numUsers` would need to be replaced by `this.numClients`